src/helpers/networkTypes.ts

```diff
@@ -7,6 +7,7 @@ export enum Network {
   polygon = 'polygon',
   rinkeby = 'rinkeby',
   ropsten = 'ropsten',
+  xdai = 'xdai',
```

Comment:
U sure this isn't gonna show up anywhere else? WC for example

Reply:
second thought we dont need atm